package.json

```diff
@@ -19,6 +19,7 @@
     "dist/bootstrap-paths.d.ts.map",
     "dist/bootstrap-paths.js",
     "dist/bootstrap-paths.js.map",
+    "dist/tsconfig.json",
```

Comment:
We seem to also duplicate `dist/src` twice.

Just wondering—do we need to independently specify which of the `dist` files we want? Would it be bad to include all of `dist/`?

Reply:
Oh right. About including the whole `dist`, I tried to "optimize" the distribution and not include the `dist/test`.

Reply:
Interesting, is `dist/test` generated because the tests are written in TS?

A better way to handle this would be to add files we don't want in `.npmignore`.

Reply:
It's because we are including it into the compilation here: https://github.com/aragon/buidler-aragon/blob/master/tsconfig.json#L21

This is important to check types correction on the tests as well during the build process.